src/pages/home/HomeBlogPreview.jsx

```diff
@@ -32,22 +42,13 @@ export default function HomeBlogPreview() {
       {
         {
           mobile: (
-            <MobileBlogPreview
-              featuredPosts={featuredPosts}
-              allPosts={allPosts}
-            />
+            <MobileBlogPreview FirstPost={FirstPost} allPosts={allPosts} />
```

Comment:
```suggestion
            <MobileBlogPreview allPosts={allPosts} />
```
So I'd remove this bit that passes `FirstPost` as a prop...